package.json

```diff
@@ -45,6 +45,7 @@
     "anymatch": "^1.3.0",
     "async-each": "^0.1.6",
     "glob-parent": "^2.0.0",
+    "inherits": "^2.0.1",
```

Comment:
Let's use `require('util').inherits` — the same one as this module, except that it's not available for browsers.